var/spack/repos/builtin/packages/git/package.py

```diff
@@ -206,6 +210,22 @@ class Git(AutotoolsPackage):
     depends_on('m4',       type='build')
     depends_on('tk',       type=('build', 'link'), when='+tcltk')
 
+    @classmethod
+    def determine_version(cls, exe):
+        output = Executable(exe)('--version', output=str)
```

Comment:
@alalazo made this suggestion on another PR:
```suggestion
        output = Executable(exe)('--version', output=str, error=str)
```
That way even if we pick up the wrong package, users don't see error messages on the command-line.